CRM/Core/Payment/Form.php

```diff
@@ -360,6 +360,15 @@ public static function validateCreditCard($values, &$errors, $processorID = NULL
    * @param bool $reverse
    */
   public static function mapParams($id, $src, &$dst, $reverse = FALSE) {
+    // Set text version of state & country if present.
```

Comment:
Per my comments on https://github.com/civicrm/civicrm-core/pull/9399#issuecomment-263537094, I don't really think that removing this part will introduce any risk as it was moved from `Confirm.php` to here fixing some e-notices for country and state. 

What makes me think is it has also been removed from `Confirm.php` now. But I've tested on the confirm and thankyou page and I didn't notice any errors. Also it has been tested by @seamuslee001, so I think we are good to go with the removal.